src/Avalonia.Diagnostics/Diagnostics/ViewModels/ViewModelBase.cs

```diff
@@ -1,34 +1,42 @@
-﻿// Copyright (c) The Avalonia Project. All rights reserved.
-// Licensed under the MIT license. See licence.md file in the project root for full license information.
-
+﻿using System;
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Runtime.CompilerServices;
-using JetBrains.Annotations;
 
 namespace Avalonia.Diagnostics.ViewModels
 {
-    internal class ViewModelBase : INotifyPropertyChanged
+    public class ViewModelBase : INotifyPropertyChanged
```

Comment:
Should be `internal`. We will end up with people using this as their base VM again.

Reply:
Arrgh? How did that come back?